test/framework/cluster.go

```diff
@@ -1729,6 +1724,75 @@ func (e *ClusterE2ETest) ValidateClusterState() {
 	}
 }
 
+// ApplyPackageFile is applying a package file in the cluster.
+func (e *ClusterE2ETest) ApplyPackageFile(packageName string, targetNamespace string, PackageFile []byte) {
+	ctx := context.Background()
+	packageMetadatNamespace := fmt.Sprintf("%s-%s", "eksa-packages", e.ClusterName)
+
+	e.T.Log("Apply changes to package", packageName)
+	err := e.KubectlClient.ApplyKubeSpecFromBytesWithNamespace(ctx, e.cluster(), PackageFile, packageMetadatNamespace)
+	if err != nil {
+		e.T.Fatalf("Error upgrading package: %s", err)
+		return
+	}
+	time.Sleep(30 * time.Second) // Add sleep to allow package to change state
+}
+
+// CurlEndpointByBusyBox creates a busybox pod with command to curl the target endpoint,
+// and returns the created busybox pod name.
+func (e *ClusterE2ETest) CurlEndpointByBusyBox(endpoint string, namespace string) string {
```

Comment:
With these functions refactored can we udpate the VerifyHelloPackageInstalled to also use these as well

Reply:
Updated